Resources/Prototypes/DeltaV/Catalog/VendingMachines/Inventories/pride.yml

```
@@ -17,4 +17,25 @@
        ClothingHeadHatXmasCrown: 2
        BedsheetRainbow: 2
        ClothingUniformColorRainbow: 2
-       ClothingUnderSocksCoder: 1
\ No newline at end of file
+       ClothingUnderSocksCoder: 2
+       Socks_Striped White_and_Light_Blue: 2
+       Socks_Striped_Captain_Blue_and_Gold: 1
+       Socks_Striped_Captain_Blue_and_White: 1
+       Socks_Striped_Captain_Gold_and_Blue: 1
+       Socks_Striped_Cargo: 5
+       Socks_Striped_Cargo_LO-Qm: 1
+       Socks_Striped_Engieering_Atmos: 5
+       Socks_Striped_Engineering: 5
+       Socks_Striped_Science: 5
+       Socks_Striped_Security: 5
+       Socks_Striped_Security_Jackboots: 5
+       Socks_Striped_White_and_Green: 2
+       Socks_Striped_White_and_Light_Blue_BunnySlipper: 2
+       Socks_Striped_White_and_Orange: 2
+       Socks_Striped_White_and_Purple: 2
+       Socks_Striped_White_and_Red: 2
+       Socks_Striped_White_and_Yellow: 2
+       Socks_Striped_White_and_Purple: 2
```

```suggestion
       Socks_Striped_White_and_Purple: 2
       Socks_Striped_White_and_Red: 2
       Socks_Striped_White_and_Yellow: 2
```
This entry is listed twice.